secrets/template/conf.yaml

```diff
@@ -0,0 +1,7 @@
+repository_name: name
```

Comment:
This is not a correct template, we will need here fields for GitHub app authentication. But feel free to change it later, when the functionality is actually implemented in `release-bot`.